cli/tools/fmt.rs

```diff
@@ -353,6 +353,17 @@ fn format_yaml(
   file_text: &str,
   fmt_options: &FmtOptionsConfig,
 ) -> Result<Option<String>, AnyError> {
+  let ignore_file = file_text
+    .lines()
+    .take_while(|line| line.starts_with('#'))
+    .any(|line| {
+      line.strip_prefix('#').unwrap().trim() == "deno-fmt-ignore-file"
```

Comment:
Maybe `starts_with` instead? Someone may add a comment about why they're ignoring

Reply:
Good point, I'll add a test for that as well